rag-localdata/readme.md

```
@@ -1,20 +1,53 @@
-Retrieval Augmented Generation (RAG) will be used to build applications that integrate data from custom data sources into a prompt for a generative AI model. RAG is a commonly used pattern for developing generative AI apps - chat-based applications that use a language model to interpret inputs and generate appropriate responses.
-
-Creating an Azure AI Search Index using the grounding data of whitepapers and reference books.
-/data/(*.pdf)
-
-
-RAG client app
-Using the search index, we can use the Azure OpenAI SDK to implement the RAG pattern in a client application. Output will go to OwnData*.json
-
-Python
-
+## Project Overview
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)